app/src/utils/eventId.ts

```diff
@@ -0,0 +1,61 @@
+export default class EventId {
+    public static eventIdToZoneId(eventId: number): number {
+        switch (eventId) {
+            case 1:
+                return 2;
```

Comment:
These need to use your new Zones constant

Reply:
Yes, I know. I would like to directly pull them from the API, but first we need to decide on a http request package

Reply:
#27. For now, let's use the manual zones mapping. They're not going to change any time soon.

Reply:
#28 to do the automatic zones mapping